net/autossh/src/opnsense/mvc/app/models/VerbNetworks/Autossh/Menu/Menu.xml

```diff
@@ -0,0 +1,9 @@
+<menu>
+    <VPN>
+        <Autossh VisibleName="Autossh" cssClass="fa fa-chain fa-fw">
+            <AutosshTunnels VisibleName="Tunnel Settings" url="/ui/autossh/settings" order="10"/>
```

Comment:
mostly opting for easier node names for sorting in larger menus, but so not relevant here. just a comment.